### DISCUSSION TITLE
Vercel doesn't recognised the path /about, while in localhost in perfectly work

### DISCUSSION THREAD

**Discussion by salvatorecalo** [vercel/community#1107] [Help] [answered]
Hi today I noticed that when I deploy my website on Vercel the bath in the navbar that takes me to /about doesn't work as it does when I work in localhost

website: salvatorecalo.vercel.app

Error: 404: NOT_FOUND
Code: NOT_FOUND

ID: fra1::zl559-1669823564558-54ca2e85fa4e

**Comment by swarnava** [comment, marked as answer]
Hi, I can see it's accessible now. Are you still having trouble?

<img width="921" alt="image" src="https://user-images.githubusercontent.com/1162991/204891323-c4e5014e-b5c2-4139-a2f7-8b59bbe03a01.png">
